RNReanimated.podspec

```diff
@@ -7,10 +7,17 @@ reactVersion = '0.0.0'
 begin
   reactVersion = JSON.parse(File.read(File.join(__dir__, "..", "react-native", "package.json")))["version"]
 rescue
-  reactVersion = '0.63.2'
+  reactVersion = '0.64.0'
 end
 
 rnVersion = reactVersion.split('.')[1]
+
+folly_prefix = ""
+if rnVersion.to_i >= 64
+  folly_prefix = "RCT-"
+end
```

Comment:
I think it's slightly more readable by using ternary:
```ruby
folly_prefix = rnVersion.to_i >= 64 ? "RCT-" : ""
```

Also, a small comment on why this is needed would be nice to have.